### ISSUE TITLE
Polish CHANGELOG public page typography and image layout

### ISSUE THREAD

**Issue by geclos** [latitude-dev/latitude-llm#4285]
The CHANGELOG public page does not meet visual QA standards.

Body text appears too low-contrast for accessibility. The current contrast ratio is 4.0, below the 4.5 minimum.

Screenshots also look stretched or otherwise use an incorrect aspect ratio.

![Screenshot showing the current CHANGELOG page text styling](https://uploads.linear.app/33c48903-b167-4658-8190-1713649b853f/aeee930c-45fa-4d5f-8e2c-7d8b5ce6e4fc/ad72a986-d5aa-4b2f-9c5a-af0f6a396474?signature=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwYXRoIjoiLzMzYzQ4OTAzLWIxNjctNDY1OC04MTkwLTE3MTM2NDliODUzZi9hZWVlOTMwYy00NWZhLTRkNWYtOGUyYy03ZDhiNWNlNmU0ZmMvYWQ3MmE5ODYtZDVhYS00YjJmLTljNWEtYWYwZjZhMzk2NDc0IiwiaWF0IjoxNzg1MjU1MDYzLCJleHAiOjE4MTY4MjU2MjN9.fPFAGBNIWuqeAXwtVDHHJwOuhGvrYhSj2oO4kgIIL74)

![Accessibility inspector showing the current text contrast ratio](https://uploads.linear.app/33c48903-b167-4658-8190-1713649b853f/584f469e-03e4-46dd-9c80-9ccfef6ed603/cf8307e7-509a-408d-8a93-66ae343c7feb?signature=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwYXRoIjoiLzMzYzQ4OTAzLWIxNjctNDY1OC04MTkwLTE3MTM2NDliODUzZi81ODRmNDY5ZS0wM2U0LTQ2ZGQtOWM4MC05Y2NmZWY2ZWQ2MDMvY2Y4MzA3ZTctNTA5YS00MDhkLThhOTMtNjZhZTM0M2M3ZmViIiwiaWF0IjoxNzg1MjU1MDYzLCJleHAiOjE4MTY4MjU2MjN9.27NAt7ZzQln_P0fQo8KhTgqgPD80vbYLWaYrX5irRiw)

![Screenshot showing the image aspect ratio issue](https://uploads.linear.app/33c48903-b167-4658-8190-1713649b853f/4fb62d54-4993-4760-a295-2570015a9194/67d7bc5e-1417-4d30-b089-f51dff9a37e2?signature=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwYXRoIjoiLzMzYzQ4OTAzLWIxNjctNDY1OC04MTkwLTE3MTM2NDliODUzZi80ZmI2MmQ1NC00OTkzLTQ3NjAtYTI5NS0yNTcwMDE1YTkxOTQvNjdkN2JjNWUtMTQxNy00ZDMwLWIwODktZjUxZGZmOWEzN2UyIiwiaWF0IjoxNzg1MjU1MDYzLCJleHAiOjE4MTY4MjU2MjN9.JKdb7jh7z0zInX9tBNHM_Iz-OnWNC2_ao-LN7fBt464)
